modules/rules_go/metadata.json

```diff
@@ -5,16 +5,6 @@
             "email": "fabian@meumertzhe.im",
             "github": "fmeum",
             "name": "Fabian Meumertzheim"
-        },
-        {
-            "email": "zplin@uber.com",
-            "github": "linzhp",
-            "name": "Zhongpeng Lin"
-        },
-        {
-            "email": "tfrench@uber.com",
-            "github": "tyler-french",
-            "name": "Tyler French"
```

Comment:
Was this intentional? You may want to add these maintainers into your `.bcr/metadata.template.json` to they don't get overwritten.

Reply:
@tyler-french Could you send the change you made to the BCR to rules_go itself?